include/crm/common/resources_internal.h

```diff
@@ -70,6 +70,111 @@ enum pcmk__restart {
     pcmk__restart_ignore,
 };
 
+//! Resource scheduling flags
+enum pcmk__rsc_flags {
+    // No resource flags set (compare with equality rather than bit set)
```

Comment:
Not important but all these can have the doxygen `!` in the front of the comments